components/forms/Button/Button.tsx

```diff
@@ -48,6 +52,7 @@ export const Button = (
       data-testid={testid ? testid : "button"}
       disabled={disabled}
       {...ariaProps}
+      {...(dataAttrName && dataAttrValue ? { [`data-${dataAttrName}`]: dataAttrValue } : "")}
```

Comment:
I would prefer we use `...props` at this point vs adding more props to the args.

We'll be looking to re-factor the Button component in the future to remove some of these "extra" props i.e. shape that have been added.

We'll also look to remove standard props that a html button should just handle like data attributes, disabled etc...

Reply:
I think there are 2 Button components (there might be a purpose to having two? not sure), the other one also had this [multiple extra prop issue](https://github.com/cds-snc/platform-forms-client/blob/develop/components/globals/Buttons/Button.tsx).

I modified the PR to remove extras, let me know if that works or if it needs changing. Thanks @timarney!

Reply:
Right --- yes there are 2 buttons components 

The "original" and the one being used on the page your modifying is the button for "public forms" vs the other one that we don't need to modify for this PR is a button we created for use within the form builder / admin.  At the time we didn't want to alter anything with "public forms".  At some point once we get some styling sorted out we can re-combine but that's way outside the scope of this PR :)